### PR TITLE
Actually use the value of the CLI argument to define the number of processes to be spawned.

### PR DESCRIPTION
**Pull request by QuentinBarrand**
Also change the name of the said argument from "--cpu" to "--processes" which is more accurate.

### CODE REVIEW

File: bin/koa-cluster [hunk 1]
```diff
@@ -28,8 +28,15 @@ cluster.setupMaster({
   args: [requirename],
 })
 
-var cpus = require('os').cpus().length
-var procs = Math.ceil(0.75 * cpus)
+var procs;
+
+if(program.processes) {
```
Comment (coderhaoxin) [inline bin/koa-cluster]: +1 

just need a `space` :)

Comment (QuentinBarrand) [inline bin/koa-cluster]: Sorry. Fixed in 56c9f72.

**Comment by coderhaoxin** [comment]
@jonathanong squash and merge this? It's a patch.

**Comment by jonathanong** [comment]
let me add you @coderhaoxin i don't use this module right now.

**Comment by coderhaoxin** [comment]
@QuentinBarrand 0.2.2

**Pull request closed without merging**